.github/workflows/helm-unit-test.yaml

```diff
@@ -19,8 +19,14 @@ jobs:
       - name: Set up helm unit test plugin
         run: helm plugin install https://github.com/helm-unittest/helm-unittest --version=0.3.0
 
-      - name: Bundle sysdig-deploy dependencies
-        run: helm dependency build ./charts/sysdig-deploy
+      - name: Bundle chart dependencies
+        run: |
+          helm dependency build ./charts/admission-controller
```

Comment:
Maybe we can add a new target in the `Makefile` which execute it on every chart?
```Makefile
deps:
	find ./charts -name "Chart.yaml" | \
		xargs -L1 dirname | \
		xargs -I% sh -c \
			"helm dependency build %"
```